### ISSUE TITLE
Examples on README don't work

### ISSUE THREAD

**Issue by illandan** [intolerable/reddit#18]
Hi,

I just tried running your examples as listed on the `README` page. And.... they didn't work
```
> ghc --version
The Glorious Glasgow Haskell Compilation System, version 7.8.4
```

So, I made some minor changes and put them up [in a gist here][1]. Hopefully, they are all fine :-)

Great work, BTW :sparkles: 

[1]: https://gist.github.com/illandan/7c76a6404c8ea510fe45

**Comment by intolerable** [comment]
they work fine with OverloadedStrings enabled, so i might put a note up about that

**Comment by illandan** [comment]
Ah, `OverloadedStrings` -- I didn't realise that :-)
Please feel free to close this issue if that fixes it. Thanks!

**Comment by intolerable** [comment]
fixed in adc975de325edf818ad9ef642c75ad76d2390d2c
